recipes/codan/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.2" %}
+
+package:
+  name: codan
+  version: {{ version }}
+
+source:
+  url: https://github.com/pedronachtigall/CodAn/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
Is the source code for the single compiled binary I see in there not available anywhere? We normally just compile those as part of the recipe.

Reply:
As I mentioned in the last comment.
We keep a version compiled to Linux dist, in the bin folder, and a version compiled to macOS in the folder "for_MacOS_users".
That the reason we need lines 6,7 and 8 to be uncommented.
During the installation tests, the build did not copy it, we need to have these commands in the post-link.sh script.

Reply:
We don't need the build part in requirements added in the meta.yaml file.
```
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```